### ISSUE TITLE
Tutorial 1.1 Quiz

### ISSUE THREAD

**Issue by adivijaykumar** [gw-odw/odw-2020#5]
The last question in the quiz for Tut 1.1 asks for a URL to the GW170817 data. 

The following code 
```
from gwosc.locate import get_event_urls
urls = get_event_urls('GW170817', duration=4096, detector='V1')
print(urls)

```
returns https://www.gw-osc.org/catalog/GWTC-1-confident/data/GW170817/L-L1_GWOSC_4KHZ_R1-1187008867-32.hdf5 as the link while the 'correct' link given in the quiz is https://www.gw-openscience.org/eventapi/json/GWTC-1-confident/GW170817/v3/V-V1_GWOSC_4KHZ_R1-1187006835-4096.hdf5.

cc - @camurria, @jkanner

**Comment by jkanner** [comment]
I think Agata's catch of different versions of GWOSC resolves this issue.  Using an updated version of the gwosc python package (0.5.3) should give the correct link, which matches what's in the quiz.

**Comment by jkanner** [comment]
Having said that, the quiz supports multiple correct answers.  @adivijaykumar I recommend you update the quiz so that both answers will be marked "correct".

**Comment by camurria** [comment]
yes, I think it should be due to the gwosc version. I have the version 0.5.3 and I double checked that if I run the python code @adivijaykumar copied, I get the link: https://www.gw-openscience.org/eventapi/json/GWTC-1-confident/GW170817/v3/V-V1_GWOSC_4KHZ_R1-1187006835-4096.hdf5
I agree that we should add both answers to be conservative.
